ui/src/app/pages/data/concept/concept-set-details.tsx

```diff
@@ -264,6 +264,14 @@ export const ConceptSetDetails = fp.flow(withUrlParams(), withCurrentWorkspace()
       navigateByUrl('workspaces/' + workspace.namespace + '/' + workspace.id + '/data/concepts' + queryParams);
     }
 
+    get displayDomainName() {
+      const {conceptSet} = this.state;
+      return conceptSet.domain === Domain.PHYSICALMEASUREMENT
+          ? 'Physical Measurements' :
+          conceptSet.domain === Domain.SURVEY
+              ? conceptSet.domain.toString() + ' - ' + conceptSet.survey : conceptSet.domain.toString() ;
```

Comment:
capitalize is only changing the case of the first letter of first word, i am using 
fp.startCase(fp.toLower(conceptSet.domain.toString())) + ' - ' + fp.startCase(fp.toLower(conceptSet.survey.toString()))
              now and have updated the screen shot to match what you have.
Also the reason i have to separate out for domain as well as survey is because otherwise i was loosing '-'